cg/meta/demultiplex/utils.py

```diff
@@ -209,14 +211,15 @@ def is_file_relevant_for_demultiplexing(file: Path) -> bool:
 def is_syncing_complete(source_directory: Path, target_directory: Path) -> bool:
     """Returns whether all relevant files for demultiplexing have been synced from the source to
     the target."""
-    manifest_file = Path(source_directory, DemultiplexingDirsAndFiles.OUTPUT_FILE_MANIFEST)
-    if not manifest_file.exists():
-        LOG.debug(
-            f"{source_directory} does not contain a "
-            f"{DemultiplexingDirsAndFiles.OUTPUT_FILE_MANIFEST} file. Skipping."
-        )
+    manifest_files: list[Path] = [
+        Path(source_directory, DemultiplexingDirsAndFiles.OUTPUT_FILE_MANIFEST),
+        Path(source_directory, DemultiplexingDirsAndFiles.CUSTOM_OUTPUT_FILE_MANIFEST),
+    ]
+    existing_files: list[Path] = [file for file in manifest_files if file.exists()]
```

Comment:
It is bit confusing to get a list and just use the first one - then why a list? Perhaps s get_manifest_file function that return the file path directly if it exists from a loop

Reply:
Same suggestion as SA and SD. Implemented 👍